modules/photosynthesis/R/plots.R

```diff
@@ -1,9 +1,8 @@
-##' @title shaded confidence interval
-##' @name ciEnvelope
-##' @author Mike Dietze
-##' @export
+## @title shaded confidence interval
+## @name ciEnvelope
+## @author Mike Dietze
```

Comment:
For cases like this where there's already a reasonably complete Roxygen block, I suggest keeping them as Roxygen comments and replacing `@export` with `@noRd` instead of converting the whole block to plain comments.

This is a reversal from what I suggested in PEcAn.logger, but here's my reasoning: It's a smaller and easier change, and it makes it explicit that the function is intentionally not exported or documented, whereas if converting to a plain comment someone might later see `# @param foo` and think "Oh, obviously that was supposed to be a Roxygen comment, let me fix that..."
```suggestion
##' @title shaded confidence interval
##' @name ciEnvelope
##' @author Mike Dietze
##' @noRd
```

Reply:
would '/document' still create a .Rd file for ciEnvelope?

Reply:
Not with the `@noRd` tag present. If we were to remove the `@export` without adding `@noRd`, it would still create an Rd file -- which is sometimes desirable for cases where you want to document functions while keeping them internal, though not what we want in this case.